.github/workflows/main.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 defaults:
   run:
-    shell: bash --noprofile --norc -o pipefail {0}
```

Comment:
Why remove this?

Reply:
Going back to the default options of GH actions

Reply:
Okay. I thought this was the default one